Apps/Playground/CMakeLists.txt

```diff
@@ -175,6 +175,15 @@ if(APPLE)
             XCODE_ATTRIBUTE_CLANG_ENABLE_OBJC_ARC YES
             RESOURCE "${RESOURCE_FILES}"
             FOLDER "Playground")
+
+        # The NSDocumentRevisionsDebugMode argument is typically added automatically by Xcode 
```

Comment:
In the PR description, you said "It looks like an old issue (10+ years) and never seen before with BN." Are there some links you could add to these comments that have more context?

Reply:
Not really. All threads I've read are the same: arguments are added automatically and it breaks command line arguments parsing. I still have no explanation on why it only happens now.